pages/prachi-chauhan.md

```diff
@@ -0,0 +1,26 @@
+## **Prachi Chauhan**
+Dallas/ CST-CDT <br/>
+MAC OSX version:10.12.3
+
+## **Biography**
+I am a Masters student at The University of Texas at Dallas since fall 2016. I completed my undergraduate in Computer Science from Indian School of Mines, Dhanbad,INDIA, in MAY 2016. So far though my educational period i have worked on various platforms and frameworks that use various new technologies:
+* STRUTS2 framework
+* Django-python framework 
+* R studio, etc.
+Here at UTD, I have taken up the following courses so far:
+1. Object Oriented Analysis and Design
+2. Statistical Methods of Data Science and Desgin 
+3. Analysis of Computer Algorithms
+4. Database Desgin
+5. Machine Learning
+6. Web Programming Languages.
+I will graduate in the spring of 2018.
+
+
+Should you have any queries on my profile, please feel free to reach out to me via email or phone:
+* prachichauhan777@gmail.com
+* pxc163630@utdallas.edu
+* contact number: (972)602-5071
+* linked in profile: <https://www.linkedin.com/in/prachi-chauhan-utd20162018/>
```

Comment:
Make this a markdown link so that the text Linkedin profile is the link like this [Linkedin profile](https://www.linkedin.com/in/prachi-chauhan-utd20162018)